.github/workflows/wiki-sync.yml

```diff
@@ -0,0 +1,52 @@
+name: Docs/Wiki Sync
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "docs/wiki/**"
+
+env:
+  wiki_source_repo: "Azure/terraform-azurerm-caf-enterprise-scale"
+  wiki_source_repo_dir: "Azure/terraform-azurerm-caf-enterprise-scale/docs/wiki"
```

Comment:
I think I'd prefer this to be just the directory within the source repo, and not include the source repo name. We could then combine these values when needed to determine the path needed.

That said, I'd prefer consistency with what has already been pushed to the ES repo so not one to worry about.

Reply:
Agreed. Maybe something for v2.